src/kili/mutations/plugins/__init__.py

```diff
@@ -123,14 +123,16 @@ def delete_plugin(
     @typechecked
     def update_plugin(
         self,
-        file_path: str,
+        plugin_path: str,
```

Comment:
Seen with you, good to have a proper error message on our end so older sdk version know they have to upgrade kili

Reply:
done!